spec/models/contributor_spec.rb

```diff
@@ -550,19 +550,28 @@
 
     describe 'when many requests are sent to the contributor' do
       before(:each) do
-        another_request = create(:request, broadcasted_at: 1.day.ago)
-        create(:message, request: the_request, recipient: contributor)
-        create(:message, request: another_request, recipient: contributor)
+        previous_request = create(:request, broadcasted_at: 1.day.ago)
+        create(:message, request: the_request, recipient: contributor, created_at: the_request.broadcasted_at)
+        create(:message, request: previous_request, recipient: contributor, created_at: previous_request.broadcasted_at)
       end
 
       it { should eq(the_request) }
     end
 
+    describe 'when most recently a manual message is sent out belonging to a previous request' do
```

Comment:
```suggestion
    describe 'when most recently a direct message is sent out belonging to a previous request' do
```

we already have manually created messages, which we are in serious discussions to remove and do not actually ever go out to the contributors. They are manually created on behalf of the contributor as a contribution that came in a different form, ie over the phone, at the market, etc. The would respond to message.sent_by_contributor? as true, but they would also have a creator_id as the user id who created it the manual message. This is in contrast to what you are doing in this test, which is to simiulate a message that was created in the chat form as a direct message toan individual contributor.